### PR TITLE
bump up finschia v2.0.0-rc6 into main

### PR DESCRIPTION
**Pull request by zemyblue**
* apply the changes of finschia v2.0.0-rc6
* fix fetch version bug of finschia-sdk and ibc in `auto-bump-up.yml` cli

### CODE REVIEW

**Review by jaeseung-bae — APPROVED** [review]
LGTM

File: .github/workflows/auto-bump-up.yml [hunk 1]
```diff
@@ -33,9 +33,9 @@ jobs:
       - name: Set each submodule version to env
         if: env.NEW_TAG
         run: |
-          FINSCHIA_SDK_VERSION=$(grep 'github.com/Finschia/finschia-sdk' go.mod | awk '{print $2}' | awk -F '-' '{if ($3 != "") {print $3} else {print $1"-"$2} }')
+          FINSCHIA_SDK_VERSION=$(grep 'github.com/Finschia/finschia-sdk' go.mod | awk '{print $2}' | awk -F '-' '{if ($3 != "" && $2 != "") {print $3} else  if ($3 != "") {print $1"-"$2} else {print $1} }')
```
Comment (tkxkd0159) [inline .github/workflows/auto-bump-up.yml:36]: it seems the second option is wrong. Do you wanna include `rc` version for that case?
Then, it should be `($3 == "")`

File: .github/workflows/auto-bump-up.yml [hunk 2]
```diff
@@ -33,9 +33,9 @@ jobs:
...
           WASMD_VERSION=$(grep 'github.com/Finschia/wasmd' go.mod | awk '{print $2}' | awk -F '-' '{if ($3 != "" && $2 != "") {print $3} else  if ($3 != "") {print $1"-"$2} else {print $1} }')
-          IBC_GO_VERSION=$(grep 'github.com/cosmos/ibc-go' go.mod | grep -v '=>' | awk '{print $2}' | awk -F '-' '{if ($3 != "") {print $3} else {print $1"-"$2} }')
+          IBC_GO_VERSION=$(grep 'github.com/cosmos/ibc-go' go.mod | grep -v '=>' | awk '{print $2}' | awk -F '-' '{if ($3 != "" && $2 != "") {print $3} else  if ($3 != "") {print $1"-"$2} else {print $1} }')
```
Comment (tkxkd0159) [inline .github/workflows/auto-bump-up.yml:38]: Same above

File: .github/workflows/auto-bump-up.yml (hunk 1 above)
Comment (zemyblue) [reply .github/workflows/auto-bump-up.yml:36]: In this case `v0.48.0-rc2.0.20230905020121-022614f80a0d`, env should be set `022614f80a0d`.
But case like `v0.48.0-rc2`, env should be set `v0.48.0-rc2`.
So I intend it.

**Pull request merged**